packages/lib/services/interop/InteropService_Exporter_Md.test.ts

```diff
@@ -446,5 +446,23 @@ describe('interop/InteropService_Exporter_Md', function() {
 		const resourceFilename = (await fs.readdir(`${exportDir()}/_resources`))[0];
 		expect(fileExtension(resourceFilename)).toBe('jpg');
 	}));
+	it.only('should replace spaces in resource name by underscores', (async () => {
```

Comment:
```suggestion
	it('should replace spaces in resource name by underscores', (async () => {
```

Reply:
I already made this change in the last commit.